app/controllers/collection_items_controller.rb

```diff
@@ -192,7 +186,7 @@ def update_multiple_with_params(allowed_items:, update_params:, success_path:)
     allowed_items.where(id: update_params.keys).each do |item|
       item_data = update_params[item.id]
       if item_data[:remove] == "1"
-        @collection_items << item unless item.destroy
+        @collection_items << item unless item.update_multiple
```

Comment:
I'm pretty sure this should still be `destroy`, which doesn't refer to the controller action, but rather to  the method for destroying the object -- which is what we want to do when removing a collection item.